components/NameTagForm.tsx

```diff
@@ -25,22 +25,26 @@ export function NameTagForm({
   content,
   onNameTagContentChange
 }: NameTagProps) {
-  const { register, handleSubmit, control } = useForm<NameTagContent>();
-
+  const { register, handleSubmit, control, watch } = useForm<NameTagContent>();
+  const maxDisclosureLength = 25;
```

Comment:
Double check the 25 character limit is a reasonable one. Maybe add some screenshots.